app/javascript/app/components/country-ghg-emissions/country-ghg-emissions-component.jsx

```diff
@@ -70,30 +75,32 @@ class CountryGhgEmissions extends PureComponent {
         </div>
         <div className={styles.graph}>
           {loading && <Loading light className={styles.loader} />}
-          {!loading && (
+          {!loading && data && data.length > 0 && config &&
             <ChartComponent
               config={config}
               data={data}
               height="100%"
               onMouseMove={handleYearHover}
               points={quantifications}
             />
-          )}
-        </div>
-        <div className={styles.tags}>
-          {config.columns &&
-            config.columns.y.map(column => (
-              <Tag
-                className={styles.tag}
-                key={`${column.value}`}
-                data={{
-                  color: config.theme[column.value].stroke,
-                  label: column.label,
-                  id: column.value
-                }}
-              />
-            ))}
+          }
+          {!loading && !data.length > 0 &&
```

Comment:
Could we keep the add Icon when the selected data is empty? Not sure if someone is going to do but the fact that if you delete all the filters you cant go back unless you go back in history or modify the URL it is annoying.

Reply:
Actually is just annoying in ESP as in GHG you can select them from the dropdown

Reply:
That was the desired behaviour but maybe it is not working. I will review.